packages/user-service/.env.example

```diff
@@ -14,8 +14,8 @@ DB_NAME=db_name
 # DB_USER=
 # DB_PASSWORD=
 
-## Cookie Parser
-COOKIE_PARSER=LDAP_SECRET
+## Certificate Configuration
+# Copy the SSL Certificate and Key path to the SSL_KEY variable and SSL_CERT directory.
```

Comment:
Double hashes so it doesn't get accidentally uncommented.

```suggestion
## Copy the SSL Certificate and Key path to the SSL_KEY variable and SSL_CERT directory.
```

Reply:
Updated this